flink-kubernetes/src/test/java/org/apache/flink/kubernetes/KubernetesClusterDescriptorTest.java

```diff
@@ -47,19 +55,29 @@
  * Tests for the {@link KubernetesClusterDescriptor}.
  */
 public class KubernetesClusterDescriptorTest extends KubernetesTestBase {
-
+	private static final String MOCK_SERVICE_HOST_NAME = "mock-host-name-of-service";
 	private static final String MOCK_SERVICE_IP = "192.168.0.1";
 
 	private final ClusterSpecification clusterSpecification = new ClusterSpecification.ClusterSpecificationBuilder()
 		.createClusterSpecification();
 
+	@Before
+	public void setup() throws Exception {
+		super.setup();
+
+		mockRestServiceWithLoadBalancer(MOCK_SERVICE_HOST_NAME, MOCK_SERVICE_IP);
+	}
+
 	@Test
 	public void testDeploySessionCluster() throws Exception {
+		mockRestServiceWithLoadBalancer(MOCK_SERVICE_HOST_NAME, MOCK_SERVICE_IP);
```

Comment:
`mockRestServiceWithLoadBalancer` could be removed. It has been called in the `setup`. Other tests is in the same situation.

Reply:
Fixed by [e55c0d2](https://github.com/apache/flink/pull/11233/commits/e55c0d2867d9c79b977bd57997e28ecb2310a616)